app/_components/ClinicCard.tsx

```diff
@@ -2,12 +2,19 @@ import Image from "next/image";
 import React from "react";
 import { BreadCrumTypes } from "../types";
 import { StarFC } from "../_atoms/Icons";
+import { getClinicsByName } from "../_utils/GlobalApi";
+import { Clinic } from "../apiTypes";
+
+// import { Clinic } from "../apiTypes"; // Arayüzü buradan import edin
+
+const ClinicCard: React.FC<BreadCrumTypes> = async ({ clinicName }) => {
```

Comment:
async function declaration main componente yapilmamali . reacz query yada useeffect  alternative solution.

Reply:
Hocam bu konuyu toplantıda tartıştık da Next'te bu bir sorun değil diye biliyoruz. Aşağıda döküman linki var. Yanlış anlaşılma varsa bakalım yine.

https://nextjs.org/learn/dashboard-app/fetching-data#fetching-data-for-revenuechart